test/jasmine/tests/dragelement_test.js

```diff
@@ -122,6 +122,19 @@ describe('dragElement', function() {
         expect(countCoverSlip()).toEqual(0);
     });
 
+    it('should not add a cover slip div to the DOM when right click', function() {
+        function countCoverSlip() {
+            return d3.selectAll('.dragcover').size();
+        }
+
+        var options = { element: this.element, gd: this.gd };
+        dragElement.init(options);
+
+        options.element.onmousedown({button: 2})
```

Comment:
Not sure, that it's a best way to emulate right click, however I can't see any other options.

Reply:
looks like in [mouse_event](https://github.com/plotly/plotly.js/blob/f1faa2871dbf182847579166af47d1cc2a983707/test/jasmine/assets/mouse_event.js#L11) we have `buttons` but if we add `button` the same way it should allow right-click simulation?